src/System.Management.Automation/engine/Subsystem/PredictionSubsystem/ICommandPredictor.cs

```diff
@@ -36,7 +36,7 @@ public interface ICommandPredictor : ISubsystem
         /// <param name="client">Represents the client that initiates the call.</param>
         /// <param name="feedback">A specific type of feedback.</param>
         /// <returns>True or false, to indicate whether the specific feedback is accepted.</returns>
-        bool CanAcceptFeedback(PredictionClient client, PredictorFeedbackKind feedback);
+        bool CanAcceptFeedback(PredictionClient client, PredictorFeedbackKind feedback) => false;
```

Comment:
It's really nice that you can do this and it doesn't mess with the polyfill for 5.1 at all. Makes sense as the signatures are probably unchanged but still neat!

Reply:
The interface default implementation is really handy 😄